lib/public/views/Logs/LogsModel.js

```diff
@@ -96,36 +96,31 @@ export class LogsModel extends Observable {
     }
 
     /**
-     * Parse the input to an array of numbers
-     * @param {string|string[]} input the input to parse
-     * @return {number[]} the parsed numbers
+     * Parse a comma-separated string of numbers to an array of numbers
+     * @param {string} commaSeparatedString The comma-separated string of numbers to parse
+     * @return {number[]} The parsed numbers
      */
-    _parseNumbers(input) {
-        // Check if the input is an array
-        if (Array.isArray(input)) {
-            return input; // Return as is if it's already an array
-        }
-        // If it's a string, split by comma, convert each part to an integer, and return the array
-        return input.split(',').map((number) => parseInt(number, 10));
+    _parseCommaSeparatedNumbers(commaSeparatedString) {
+        return commaSeparatedString.split(',').map((numberString) => parseInt(numberString.trim(), 10));
     }
 
     /**
      * Parse the current route parameters to load the log creation model
      *
      * @param {object} queryParams the current query parameters
-     * @param {string|string[]} [queryParams.runNumbers] the run numbers to link to the log being created
-     * @param {string|string[]} [queryParams.lhcFillNumbers] the lhc fill numbers to link to the log being created
-     * @param {string|string[]} [queryParams.environmentIds] the environment ids to link to the log being created
+     * @param {string} [queryParams.runNumbers] the run numbers to link to the log being created
+     * @param {string} [queryParams.lhcFillNumbers] the lhc fill numbers to link to the log being created
+     * @param {string} [queryParams.environmentIds] the environment ids to link to the log being created
      * @return {void}
      */
     loadCreation({ runNumbers, lhcFillNumbers, environmentIds }) {
         const relations = {};
 
         if (runNumbers) {
-            relations.runNumbers = this._parseNumbers(runNumbers);
+            relations.runNumbers = this._parseCommaSeparatedNumbers(runNumbers);
         }
         if (lhcFillNumbers) {
-            relations.lhcFillNumbers = this._parseNumbers(lhcFillNumbers);
+            relations.lhcFillNumbers = this._parseCommaSeparatedNumbers(lhcFillNumbers);
         }
         if (environmentIds) {
             relations.environmentIds = Array.isArray(environmentIds) ? environmentIds : [environmentIds];
```

Comment:
If you change the docblock, this should be changed. If you can't have an array, you should probably actually split the environmentIds here too (no conversion to number though)